packages/infra/lib/hl7-notification-stack/index.ts

```diff
@@ -4,34 +4,24 @@ import { Repository } from "aws-cdk-lib/aws-ecr";
 import { Construct } from "constructs";
 import { EnvConfigNonSandbox } from "../../config/env-config";
 import { MetriportCompositeStack } from "../shared/metriport-composite-stack";
+import { HL7_NOTIFICATION_VPC_CIDR } from "./constants";
 import { MllpStack } from "./mllp";
 import { NetworkStack } from "./network";
-import { VpnStack } from "./vpn";
-import { INTERNAL_SERVICES_SUBNET_GROUP_NAME } from "./constants";
-import { VPN_ACCESSIBLE_SUBNET_GROUP_NAME } from "./constants";
-import { Secret } from "aws-cdk-lib/aws-secretsmanager";
 
 export interface Hl7NotificationStackProps extends cdk.StackProps {
   config: EnvConfigNonSandbox;
   version: string | undefined;
 }
 
-const NUM_AZS = 2;
-
-const fetchSecretsForPartner = (scope: Construct, partnerName: string) => {
-  const secretName = `PresharedKey-${partnerName}`;
-  return Secret.fromSecretNameV2(scope, secretName, secretName);
-};
+const NUM_AZS = 1;
```

Comment:
Couldn't we have 2 AZs and, if the partner HIE only supports a single IP, we send the main AZ's?

Reply:
We could, but I don't believe _any_ of these HIEs will support outbound to 2 destinations. I wanted to keep the infra easy to understand and iterate upon, as I'll soon need to set up a transit gateway (v1 of project) at which point I'll switch to multi AZ and update the route tables to map the IP of the single NLB to our multi-az NLB.